### DISCUSSION TITLE
Could not find a Cypress configuration file when trying to run tests

### DISCUSSION THREAD

**Discussion by Belf33** [Synthetixio/synpress#520] [General]
I have updated Synpress from 1.2 -> 2.3.3 version and framework cant start

![image](https://user-images.githubusercontent.com/20465760/192770980-c5c3e746-ab27-44e4-af16-2812be272df0.png)

change only version of Synpress

here whole project
https://github.com/Belf33/Synpress_web3_UI_framework_mvp/tree/newVershion

may be someone faced that problem?

**Comment by drptbl** [comment]
Please follow the readme. Your config file is still in old format (json).

**Reply by Belf33** [reply]
Ok, I got you. I rebuilt all, and I faced another issue. maybe you faced that before
https://github.com/Synthetixio/synpress/discussions/524
